.github/workflows/arm_deb_packager.yml

```
@@ -26,7 +26,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@master
```

should we use v5 here as well, wdyt?
```suggestion
        uses: actions/setup-go@v5
```